src/sensors/TFAxes.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @author Jihoon Lee - jihoon.lee@kakaobrain.com
+ */
+/**
+ * An Axes node can be used to display the axis of a particular coordinate frame.
+ *
+ * @constructor
+ * @param options - object with following keys:
+ *
+ *   * frame_id - the frame id to visualize axes
+ *   * tfClient - the TF client handle to use
+ *   * shaftRadius (optional) - the radius of the shaft to render
+ *   * headRadius (optional) - the radius of the head to render
+ *   * headLength (optional) - the length of the head to render
+ *   * scale (optional) - the scale of the frame (defaults to 1.0)
```

Comment:
It seems like scale, lineType, and lineDashLength are not used.

Reply:
Addressed.